helm-charts/ingress-controller/templates/Deployment.yaml

```diff
@@ -23,10 +22,11 @@ spec:
         - -c
         - |-
           echo name: $EG_NAME > /opt/eg-config/config.yaml &&
```

Comment:
`EG_NAME` are samed with `{{ .Release.Name }}`. You can directly introduce the variable in the helm-charts/ingress-controller/templates/ConfigMap.yaml 
```
kind: ConfigMap
metadata:
  name: {{ .Release.Name }}
data: |
  easegress-server.yaml: |
    name: {{.Release.Name}}
    cluster-name: easegress-ingress-controller
    cluster-role: primary
    api-addr: 0.0.0.0:2381
    data-dir: /opt/easegress/data
    log-dir: /opt/easegress/log
    debug: false
```